assignments/arrays.js

```diff
@@ -63,34 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[33-1]['car_year']} ${inventory[33-1]['car_make']} ${inventory[33-1]['car_model']}`);
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1];
+console.log(`Make of last car is ${lastCar.car_make} and model is ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = inventory.sort(function(a, b) {
+    let x = a.car_model.toLowerCase();
+    let y = b.car_model.toLowerCase();
+    if(x < y) { return -1};
+    if(x > y) { return 1};
+    return 0;
+  });
+  for (let i = 0; i < carModels.length; i++) {
+      let eachSortedCar = carModels[i];
+      console.log(eachSortedCar.car_model);
+      }
+  
```

Comment:
I really like how you are making use of `sort()` and `toLowerCase()`. Another example of how you could have solved this would be 
```let carModels = [];
for (let i = 0; i < inventory.length; i++) {
  carModels.push(inventory[i].car_model);
}
console.log(carModels.sort());